openmdao/docs/features/building_blocks/components/interp.rst

```diff
@@ -0,0 +1,75 @@
+.. index:: BsplinesComp Example
+
+.. _bsplinescomp_feature:
+
+************
+BsplinesComp
+************
+
+The BsplinesComp allows you to represent a large dimension variable as a smaller dimensional variable via a B-spline function.
+This is useful for reducing the size of an optimization problem by decreasing the number of design variables it solves, and potentially
+ithe number of gradients it computes if you are running in forward mode. You may consider using this when you have spatially
+distributed design variables where you know that the physics gives you smooth designs that can be well approximated by B-splines.
+
+
+BsplinesComp Example
+--------------------
+
+The following example shows the optimization of a simple beam with rectangular cross section. The beam has been subdivided into
+elements along the length of the beam, and one end is fixed. The goal is to minimize the volume (and hence the mass of the
+homogeneous beam) by varying the thickness in each element without exceeding a maximum stress constraint while the beam is
+subject to multiple load cases, each one being a distributed force load that varies sinusoidally along the span.
+
+If we allow the optimizer to vary the thickness of each element, then we have a design variable vector that is as wide as the
+number of elements in the model. This may perform poorly if we have a large number of elements in the beam. If we assume that
+the optimal beam thickness is going to have a smooth continuous variation over the length of the beam, then it is a good
+candidate for using an interpolation component like BSplinesComp.
```

Comment:
small detail, but capitalization is diff from class for BSplinesComp here and 2nd line below

Reply:
All items fixed.